tests/ClientFunctionalTest.php

```diff
@@ -97,4 +97,12 @@ private function assertDatabaseExists($databaseName, $callback = null)
             call_user_func($callback, $foundDatabase);
         }
     }
+
+    public function testStartSession()
+    {
+        if (version_compare($this->getFeatureCompatibilityVersion(), '3.6', '<')) {
+            $this->markTestSkipped('startSession() is only supported on FCV 3.6 or higher');
+        }
+        $session = $this->client->startSession();
```

Comment:
`$this->assertInstanceOf('MongoDB\Driver\Session', $this->client->startSession())`

Reply:
The assertion below is now creating a new object, so the `$session` variable assigned above is no longer used and can be deleted.